src/ert/scheduler/openpbs_driver.py

```diff
@@ -50,12 +49,16 @@ def __init__(
         queue_name: Optional[str] = None,
         memory_per_job: Optional[str] = None,
         job_prefix: Optional[str] = None,
+        num_retries: int = 10,
```

Comment:
I am not sure we need to expose these as class parameters unless we intend to have it adjustable through the configuration.

For testing it should be ok to modify class internal variables, so instead of 
```
driver = OpenPBSDriver(num_retries=2)
```
we can do
```
driver = OpenPBSDriver()
driver._num_retries = 2
```

Reply:
> I am not sure we need to expose these as class parameters unless we intend to have it adjustable through the configuration.

That's the point, do we want to have it exposed or not? Maybe not - if we decide to have there I'll exclude it also `retry_interval`.